website/screens/components/application-layout/usage/ApplicationLayoutUsagePage.tsx

```diff
@@ -2,77 +2,129 @@ import {
+          The application layout is the base component that wraps any
+          application built with Halstack. In addition to this, all of the
+          components listed below can be used within each other to create a wide
+          variety of standard layouts:
         </DxcParagraph>
+        <DxcBulletedList>
+          <DxcBulletedList.Item>Flex</DxcBulletedList.Item>
+          <DxcBulletedList.Item>Bleed </DxcBulletedList.Item>
+          <DxcBulletedList.Item>Inset</DxcBulletedList.Item>
+        </DxcBulletedList>
+      </>
+    ),
+  },
+  {
+    title: "Grid",
+    content: (
+      <>
+        <DxcParagraph>
+          The grid provides the foundation for consistently positioning elements
+          onscreen. The 8px Grid is the geometric foundation of all the visual
+          elements of Halstack Design System components and spacing. It provides
+          structure and guidance for all creative decision-making.
+        </DxcParagraph>
+        <Image src={layoutGrid} alt="Layout grid of 8px" />
         <DxcBulletedList>
           <DxcBulletedList.Item>
-            Use the sidenav when the application navigation has an horizontal
-            hierarchy
+            Use multiples of 8px when defining measurements, spacing, and
+            positioning elements.
           </DxcBulletedList.Item>
           <DxcBulletedList.Item>
-            Header and footer elements should be always present
+            When necessary use 4px to make more fine tuned adjustments.{" "}
           </DxcBulletedList.Item>
           <DxcBulletedList.Item>
-            Only use the mode overlay if the content cannot dynamically adapt to
-            the available space
+            Whenever possible, make sure that objects line up, both vertically
+            and horizontally.
+          </DxcBulletedList.Item>
+          <DxcBulletedList.Item>
+            Align your bounding box to the grid, not the baseline of your text.
           </DxcBulletedList.Item>
         </DxcBulletedList>
       </>
     ),
   },
   {
-    title: "Sidenav: overlay and push",
+    title: "Breakpoints",
     content: (
       <>
-        <Figure caption="Overlay and push behavior of the sidenav component">
-          <Image
-            src={SidenavBehavior}
-            alt="Overlay and push behavior of the sidenav component"
-          />
-        </Figure>
         <DxcParagraph>
-          Overlay and push interactions by the sidenav component can alter the
-          way the layout is displayed. For overlay, the sidenav simply slides
-          over the content with no changes to position. The push interaction
-          moves the main container to the side depending on the width of the
-          sidenav.
+          Breakpoints define resolutions at which screen components adjust to
+          offer an optimal user experience across screen sizes and devices.
+          We&#39;ve defined five different breakpoints to accommodate multiple
```

Comment:
Maybe change the code `&#39;` with a normal `'`.

Reply:
`&#39;` is the same as `'`, and we avoid the error that gives in Visual Studio.